### PR TITLE
fix (datatable): remove json-statham dependencies.

### PR DESCRIPTION
**Pull request by doktordirk**
json-statham won't work with jspm bundles atm. plus, it's overkill
so, i copied my code from aurelia-config in instead

@RWOverdijk  please review, merge and publish

### CODE REVIEW

File: src/datatable.js [hunk 1]
```diff
@@ -219,6 +218,27 @@ export class DataTable {
   }
 
   displayValue(row, propertyName) {
-    return new Statham(row, Statham.MODE_NESTED).fetch(propertyName);
+    return fetchFrom(row,propertyName);
```
Comment (RWOverdijk) [inline src/datatable.js]: Space after ,

**Comment by RWOverdijk** [comment]
@jeremyvergnas Please review

File: src/datatable.js [hunk 2]
```diff
@@ -219,6 +218,27 @@ export class DataTable {
   }
 
   displayValue(row, propertyName) {
-    return new Statham(row, Statham.MODE_NESTED).fetch(propertyName);
+    return fetchFrom(row, propertyName);
```
Comment (jeremyvergnas) [inline src/datatable.js]: It won't work with association columns. `propertyName` can be "car.color", and in that case `displayValue` should return `color`. Which won't be the case now I think.

Comment (doktordirk) [inline src/datatable.js]: right. an error in my docblocks, made me copy only have the needed code.

**Comment by jeremyvergnas** [comment]
👍

**Pull request merged**